pkg/storaged/block/format-dialog.jsx

```diff
@@ -196,6 +209,24 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
     if (create_partition && enable_dos_extended)
         add_fsys(true, { value: "dos-extended", title: _("Extended partition") });
 
+    function is_supported(type) {
+        return filesystem_options.find(o => o.value == type);
+    }
+
+    let default_type = null;
+    if (block.IdUsage == "filesystem" && is_supported(block.IdType))
+        default_type = block.IdType;
+    else {
+        const root_block = find_root_fsys_block();
+        console.log("ROOT", root_block);
+        if (root_block && is_supported(root_block.IdType))
+            default_type = root_block.IdType;
+        else if (client.anaconda?.default_fsys_type && is_supported(client.anaconda.default_fsys_type))
+            default_type = client.anaconda.default_fsys_type;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19854-20240118-161616-eaf4cdd4-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)